From 755365d7d722a96cbbc6b2652abee11b4bc18155 Mon Sep 17 00:00:00 2001
From: Patrick Heeb <speedpat@biochips.(none)>
Date: Mon, 17 Feb 2014 13:43:05 +0100
Subject: [PATCH] remove gtk from BUILD.gn

---
 build/config/linux/BUILD.gn |   21 +++------------------
 1 file changed, 3 insertions(+), 18 deletions(-)

diff --git a/build/config/linux/BUILD.gn b/build/config/linux/BUILD.gn
index d166ac7..140b9f7 100644
--- a/build/config/linux/BUILD.gn
+++ b/build/config/linux/BUILD.gn
@@ -38,14 +38,13 @@ pkg_config("freetype2") {
 }
 
 pkg_config("glib") {
-  packages = [ "glib-2.0", "gmodule-2.0", "gobject-2.0", "gthread-2.0" ]
+  packages = [ "freetype2" ]
 }
 
 pkg_config("gtk") {
   # Gtk requires gmodule, but it does not list it as a dependency in some
   # misconfigured systems.
-  packages = [ "gmodule-2.0", "gtk+-2.0", "gthread-2.0" ]
-  defines = [ "TOOLKIT_GTK" ]
+  packages = [ "freetype2" ]
 }
 
 pkg_config("pangocairo") {
@@ -57,19 +56,5 @@ pkg_config("udev") {
 }
 
 config("x11") {
-  # Don't bother running pkg-config for these X related libraries since it just
-  # returns the same libs, and forking pkg-config is slow.
-  defines = [ "USE_X11" ]
-  libs = [
-    "X11",
-    "Xcomposite",
-    "Xcursor",
-    "Xdamage",
-    "Xext",
-    "Xfixes",
-    "Xi",
-    "Xrender",
-    "Xss",
-    "Xtst",
-  ]
+  packages = [ "freetype2" ]
 }
-- 
1.7.10.4

